munet/base.py

```diff
@@ -940,7 +940,15 @@ def _cmd_status_finish(self, p, c, ac, o, e, raises, warn):
                 raise CalledProcessError(rc, ac, o, e)
         return rc, o, e
 
-    def _cmd_status(self, cmds, raises=False, warn=True, stdin=None, **kwargs):
+    def _cmd_status(
+        self,
+        cmds,
+        raises=False,
+        warn=True,
+        stdin=None,
+        output=True,
```

Comment:
I understand you're looking for a "fire-and-forget" command, I just wonder if this is the right way to do that.

I think that perhaps instead of `output` the new param should be `no_wait=False`.

I'm also wondering if we should explicitly set `stdout=subprocess.DEVNULL` and `stderr=subprocess.DEVNULL`.